tests/func-tests/observability_controller_test.go

```diff
@@ -55,3 +68,13 @@ var _ = Describe("Observability Controller", Label(tests.OpenshiftLabel, "observ
 		})
 	})
 })
+
+func serviceAccountTlsCertPathExists() bool {
+	_, err := os.ReadFile(observability.ServiceAccountTlsCertPath)
```

Comment:
No need to actually read the file. you can use `os.Stat()` for that; e.g.
 ```suggestion
	_, err := os.Stat(observability.ServiceAccountTlsCertPath)
```

Reply:
updated